spec/integration_spec.rb

```diff
@@ -205,10 +205,10 @@ class Comment < ActiveRecord::Base; end
 
       it 'does not allow to migrate' do
         expect do
-          ClimateControl.modify PERCONA_ARGS: '--arg=foo' do
+          ClimateControl.modify PERCONA_ARGS: '--sleep=1' do
             ActiveRecord::Migrator.migrate(migrations_paths, 1)
           end
-        end.to raise_error do |exception|
+        end.not_to raise_error do |exception|
```

Comment:
We should be able to remove the block from this, as we don't need to check an error we don't expect to be raised